src/resmom/linux/mom_mach.c

```diff
@@ -8001,6 +8008,63 @@ pidcache_create(void)
 	return (pidcache_arena);
```

Comment:
Looking at the comment I just realized we don't need pidcache_starttime anymore. On calloc failure we can return from mom_get_sample() itself.